model/src/main/proto/interaction_object.proto

```diff
@@ -57,6 +58,11 @@ message ListOfSetsOfHtmlStrings {
   repeated StringList set_of_html_strings = 1;
 }
 
+message Point2d {
```

Comment:
Please add comments for this explaining what it is?

Reply:
done